requirements.txt

```diff
@@ -1,4 +1,7 @@
+chainmap
```

Comment:
Ah, we need to call `install_aliases`, but it's [currently broken anyway](https://github.com/PythonCharmers/python-future/issues/226). Let's leave it in for now.